src/community/develop-a-component-or-pattern/index.md.njk

```diff
@@ -30,27 +29,29 @@ If you’re happy to go ahead, the team will assign you to the issue as a contri
 
 ## Research and develop your contribution
 
-While you’re working on your contribution, the Design System team will arrange a weekly catch up to find out how you’re getting on and if you need any help. 
+While you’re working on your contribution, the Design System team will arrange a weekly catch up to find out how you’re getting on and if you need any help.
 
 Find examples of the component or pattern already in use. The best way to do this is to ask the government design community on the [digital service designers mailing list](https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/?hl=en-GB#!forum/digital-service-designers) or the [#design channel on cross-government Slack](https://ukgovernmentdigital.slack.com/app_redirect?channel=design).
 
 Examples and research from government services are usually most relevant. But look at how other organisations solve the problem too.
 
-Involve people from the [government design community](https://www.gov.uk/service-manual/communities/design-community) in any work you’re doing. This makes it less likely that you’ll spend time doing work or research that’s already been done. 
+Involve people from the [government design community](https://www.gov.uk/service-manual/communities/design-community) in any work you’re doing. This makes it less likely that you’ll spend time doing work or research that’s already been done.
 
 By finding people who are working on similar solutions, you can benefit from progress they’ve made and relevant research they’ve done.
 
 Once you have collected some examples and research findings, you should start designing a specific implementation of the component or pattern to research with.
 
-Ideally you’ll research as part of an actual service, but it’s possible to test using prototypes as long as they are realistic. Work with your own team or find a team to work with through the design community. 
+Ideally you’ll research as part of an actual service, but it’s possible to test using prototypes as long as they are realistic. Work with your own team or find a team to work with through the design community.
+
+If you're making a component, you'll also need to [make sure it's accessible by testing it against both general and specific acceptance criteria](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/test-components-using-accessibility-acceptance-criteria.md).
```

Comment:
This line is the only substantive change to this page.

I know we've talked about overhauling this page. Will that work be part of this PR, or are we just adding the link for now and iterating later?

Reply:
Could we do a light touch, breaking these paragraphs into H3s in here to make it a bit easier to scan:
- Start with what exists
- Work with the community
- Test with users

I can't suggest an edit that big in here for some reason, but hope you get the idea!

Reply:
@calvin-lau-sig7 Sounds good, thanks!

Have also added 'intro' sentence (line 34) - how does that look?

Reply:
Looks good! And the new heading is good too.

Reply:
@calvin-lau-sig7 Cheers! Will squash commits